pkg/user/delivery/http/user_proxy.go

```diff
@@ -14,25 +14,40 @@ import (
 // UserProxy represents the interface to the user's proxy operations
 type UserProxy interface {
 	Create(user entities.User) (err error)
-	CreateToken(user entities.User) (string, error)
+	CreateUserToken(user entities.User) (token string, err error)
+	CreateAppToken(user entities.User) (token string, err error)
 }
 
 // Proxy is responsible for implementing the user's proxy operations
 type Proxy struct {
-	url    string
-	logger logging.Logger
+	userURL string
+	authURL string
```

Comment:
Just a suggestion :point_down: 

```suggestion
	authnURL string
```
> If accepted, please remember to update all occurrences of this variable :smile:

Reply:
OK! I gonna add a new commit changing both `authURL` to `authnURL` and `userURL` to `usersURL`. What you think?

Reply:
No problem! Looks good to me :smile:

Reply:
Since I create a new `authn_proxy.go`, I removed that prefix and let `URL`